charts/acp/templates/configmap.yaml

```diff
@@ -41,7 +41,8 @@ data:
       {{- else }}
       enabled: true
       addrs:
-      - "{{ .Release.Name }}-redis-master:6379"
+      - "{{ .Release.Name }}-redis-cluster-headless:6379"
```

Comment:
do we need double adresses?

Reply:
Yes, thats intended. We need at least 2 addresses to force client into cluster mode. With a single address client is using standalone mode. Those adresses return all IPs of acrive redis nodes